tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Is there any reason why you set the target to es5??

Reply:
The default setting was left, and I updated the target to ES2022. Thank you for catching this up!